Sources/SWBUtil/Path.swift

```diff
@@ -717,9 +744,10 @@ public struct Path: Serializable, Sendable {
             var numComponents = 0
             var isInPathComponent = false
             var nextCharacterIsEscaped = false
-            for idx in pattern.indices {
+            for byte in pattern.utf8 {
                 // Skip over path separators, unless they're escaped.
-                if pattern[idx] == Path.pathSeparator {
+                //TODO: should this (and other similar uses) be Path.isUTF8PathSeparator(byte) instead for Windows?
```

Comment:
I'd like feedback on this please. To my eye it looks like a spot where Windows would want us to check both path separators, but I wasn't sure, and the old code didn't do so.